libraries/AP_RPM/AP_RPM.h

```diff
@@ -38,11 +39,10 @@ class AP_RPM
     // RPM driver types
     enum RPM_Type {
         RPM_TYPE_NONE    = 0,
-        RPM_TYPE_PWM     = 1,
```

Comment:
better to leave the old numbers, and still map PWM to PIN, but don't list PWM in the docs. This makes it better if users manually copy params or use older docs